public/js/app/components/MemoMenu.jsx

```diff
@@ -68,7 +69,7 @@ function MemoMenu(props) {
       rounds: [],
       newSemester: true,
     })
-  }
+  }, [])
```

Comment:
Is eslint not complaining about this empty dep-array here? `getUsedRounds` should be added or be inside `useCallback`, right?